Samples/WinMLSamplesGallery/WinMLSamplesGalleryNative/Win32Application.cpp

```diff
@@ -0,0 +1,164 @@
+#include "pch.h"
+#include "stdafx.h"
+#include "Win32Application.h"
+
+HWND Win32Application::m_hwnd = nullptr;
+bool Win32Application::close_window = false;
```

Comment:
Tis a member variable like `m_hwnd`, and so `m_` prefix too, `m_closeWindow`.

Reply:
Done.